pkg/exposer/csi_snapshot.go

```diff
@@ -463,6 +464,17 @@ func (e *csiSnapshotExposer) createBackupVSC(ctx context.Context, ownerObject co
 		},
 	}
 
+	/*
+		We need to keep the label of the managing node for distributed snapshots.
+		The external snapshot manager will only manage snapshots matching it's node if that feature is enabled.
+
+		https://github.com/kubernetes-csi/external-snapshotter/tree/4cedb3f45790ac593ebfa3324c490abedf739477?tab=readme-ov-file#distributed-snapshotting
+		https://github.com/kubernetes-csi/external-snapshotter/blob/4cedb3f45790ac593ebfa3324c490abedf739477/pkg/utils/util.go#L158
+	*/
+	if manager, ok := snapshotVSC.Labels["snapshot.storage.kubernetes.io/managed-by"]; ok {
```

Comment:
Let's add a const for this label `snapshot.storage.kubernetes.io/managed-by`, in https://github.com/vmware-tanzu/velero/blob/f985879f4bf7e69b76d986d5456ebd3bf079ed4c/pkg/util/kube/utils.go#L48.